pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -87,16 +87,18 @@ def compile(
     host_install_dir: str,
     target_install_dir: str,
     replace_libs: str,
-):
+) -> None:
     ...
 
 
 @overload
-def compile(*, mypy__Single_overload_definition_multiple_required: int):
+def compile(
+    env: dict[str, str], *, mypy__Single_overload_definition_multiple_required: str
```

Comment:
This is only added because mypy requires two overloads - but it needs to be compatible with the implementation.

Reply:
Would be nice if we could get mypy to fix this. Giving typings for `**kwargs` functions is useful. Between:
```py
def f(*, arg1 : Type1, arg2 : Type2, arg3 : Type3):
   d = dict(arg1=arg1, arg2=arg2, arg3=arg3)
```
and 
```py
@overload
def f(*, arg1 : Type1, arg2 : Type2, arg3 : Type3): ...

def f(**kwargs):
   d = kwargs
```
The second requires significantly less repetition.

Reply:
The former works at runtime too, though, you can inspect the signature. The later is a rather ugly hack.

Reply:
Well if you think so I am fine with replacing it with the explicit way.

Reply:
FYI, I've used another ugly hack before:

```python
def f(*, arg1 : Type1, arg2 : Type2, arg3 : Type3):
   d = dict(**locals())
```

🤣 - It's not like this gets rewritten every day, or changes too much (I think), so the explicit, static way is likely fine.